clinica/iotools/bids_utils.py

```diff
@@ -885,6 +886,11 @@ def run_dcm2niix(
 
     bids_sidecar : bool, optional
         Whether to generate a BIDS sidecar or not. Default=True.
+
+    Returns
+    -------
+    dcm2niix_success: bool
```

Comment:
Since you're not naming the return value:

```suggestion
    bool :
```